### ISSUE TITLE
Color Contrast Not Failing - Works with react-axe

### ISSUE THREAD

**Issue by natac13** [avanslaars/cypress-axe#41]
Hey there. I am experience what I think is a strange issue. I am testing out the accessibility on my website built with Gatsby (React.js). I have Cypress set up as per their docs with this library and axe-core. When I run the the following I get no errors on a page that has a footer which I know is rendering links that will fail the color contrast test. I know this because I have react-axe installed as well and it is reporting the color contrast problem in the console. 
However because react-axe does not lead to a nice automation testing work flow I would like to use cypress to find these color contrast issues. 

What could be causing cypress-axe / axe-core not picking it up with testing but react-axe picks it up as expected? 

Sample test
```js
describe('Accessibility tests', () => {
    beforeEach(() => {
      cy.visit('http://localhost:8000')
      cy.injectAxe()
    })
    it('Has no detectable accessibility violations on load', () => {
      cy.wait(1000)
      cy.checkA11y()
    })
  })
```

**Comment by natac13** [comment]
This is my problem I think. I just tried to just to get a simple example setup to show my results, however I got the expected results of the color contrast reporting an error. Ill have to figure out what is causing my application setup to not report the contrast problem where the simple example does.

Thanks for the Package!
